templates/frontend/pages/indexJournal.tpl

```diff
@@ -89,6 +88,21 @@
 			</div>
 		{/if}
 
+		{if $announcements}
+		<section class="row homepage-announcements">
+			<h2 class="sr-only">{translate key="announcement.announcementsHome"}</h2>
+			{foreach from=$announcements item=announcement}
+				<article class="col-md-4 homepage-announcement">
+					<h3 class="homepage-announcement-title">{$announcement->getLocalizedTitle()|escape}</h3>
+					<p>{$announcement->getLocalizedDescriptionShort()|strip_unsafe_html} <br><a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}">{translate key="common.more"}…</a></p>
```

Comment:
The `...` here probably needs to be translatable for other languages. I'd suggest a theme-specific locale string like:

```
<message key="plugins.themes.healthSciences.more">{$text}...</message>
```

And then doing something like:

```
{capture var="more"}{translate key="common.more"}{/capture}
{translate key="plugins.themes.healthSciences.more" text=$more}
```

Reply:
rebased + force-pushed!